macros/edr/metadata_collection/get_tables_from_information_schema.sql

```diff
@@ -7,7 +7,7 @@
     {%- set database_name, schema_name = schema_tuple %}
     {% set schema_relation = api.Relation.create(database=database_name, schema=schema_name).without_identifier() %}
 
-    (with information_schema_tables as (
+    with information_schema_tables as (
```

Comment:
Why was this removed? 
I think it was there for a reason

Reply:
I think it was because of my comment (I didn't realize why it's needed)

Reply:
Moved to the [macro itself](https://github.com/elementary-data/dbt-data-reliability/pull/230/commits/6fc9b19c4d3d0e0ed1afe972b8795cab44350d30).